core/src/histogram.cc

```diff
@@ -17,13 +17,11 @@ Histogram::Histogram(const BucketBoundaries& buckets)
                         std::end(bucket_boundaries_)));
 }
 
-void Histogram::Observe(const double value) {
-  // TODO: determine bucket list size at which binary search would be faster
-  const auto bucket_index = static_cast<std::size_t>(std::distance(
-      bucket_boundaries_.begin(),
-      std::find_if(
-          std::begin(bucket_boundaries_), std::end(bucket_boundaries_),
-          [value](const double boundary) { return boundary >= value; })));
+void Histogram::Observe(double value) {
```

Comment:
```suggestion
void Histogram::Observe(const  double value) {
```

Could you please restore the `const`? The author uses it to show that the passed value is not modified within the function. In case we decide to remove it it should be done everywhere.

Reply:
Sure, done